### DISCUSSION TITLE
Sorting a collection by field in the CMS panel

### DISCUSSION THREAD

**Discussion by ldanielswakman** [tinacms/tinacms#3907] [Ideas]
In the Tina CMS panel, in order to ship this UI to clients, I would need to customise the list to fit the client's needs. For instance, in this example below I have an 'order' field that I want the collection to be sorted by. Does that make sense?

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/4538367/236334497-99c42805-bef4-44af-b37c-edcd4316175b.png">

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/4538367/236334621-47abf887-ff3d-44a6-8207-4cdc8b42beb7.png">

**Comment by appinteractive** [comment]
There is already a sorting method, but it's not shown when you have multiple templates in one collection. There's also a rudimental "search" (starts with) which is also hidden. 

As far as I understood, they are working on it.

**Comment by Revenciel** [comment]
Same thing for a collection! I would love there to be a visible order and a native drag-and-drop way to reorder, for example, pages in a page collection.

This would allow for automatic sorting of pages in a component (such as a nav or site map) that shows a list of all pages (or a list of all items in a collection).

**Reply by appinteractive** [reply]
> Same thing for a collection! I would love there to be a visible order and a native drag-and-drop way to reorder, for example, pages in a page collection.
> 
> 
> 
> This would allow for automatic sorting of pages in a component (such as a nav or site map) that shows a list of all pages (or a list of all items in a collection).

Hey you can accomplish this by (for example) using a json file that you fill with the pages you wish an then use them for rendering. This has a lot of advantages as you are able to filter them etc. sorting the is only for one in the backend not for output, this are two complete different thinks.
